students/kondel_tomasz/lesson_04_unit_testing/collatz_test.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+import sys
+sys.path.insert(0, '../lesson_03_funtions/')
+from the_collatz_sequence import collatz  # noqa: E402
+
+
+class MyFirstUnitTest(unittest.TestCase):
+
+    def test_assertRaises(self):
```

Comment:
With naming unit tests try to be as specific as possible, for example `CollatzTest.test_raises_ValueError_on_string`. That way you can go right away to fix the error in the code without looking at the test code.